core/domain/feature_flag_registry_test.py

```diff
@@ -0,0 +1,225 @@
+# coding: utf-8
+#
+# Copyright 2023 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for feature flag registry."""
+
+from __future__ import annotations
+
+import enum
+
+from core import feconf
+from core import utils
+from core.constants import constants
+from core.domain import feature_flag_domain
+from core.domain import feature_flag_registry as registry
+from core.tests import test_utils
+
+FeatureStages = feature_flag_domain.FeatureStages
+
+
+class FeatureNames(enum.Enum):
+    """Enum for parameter names."""
+
+    FEATURE_A = 'feature_a'
+
+
+class FeatureFlagRegistryTests(test_utils.GenericTestBase):
+    """Tests for the feature flag Registry."""
+
+    def setUp(self) -> None:
+        super().setUp()
+
+        self.original_feature_flag_spec_registry = (
+            registry.Registry.feature_flag_spec_registry)
+        registry.Registry.feature_flag_spec_registry.clear()
+
+    def tearDown(self) -> None:
+        super().tearDown()
+
+        registry.Registry.feature_flag_spec_registry = (
+            self.original_feature_flag_spec_registry)
+
+    def test_create_feature_flag(self) -> None:
+        # Here we use MyPy ignore because to test the functionalities with dummy
```

Comment:
I don't understand why we need to do this in all the tests (and I also don't really understand what the comment is saying). Can we avoid it?